## Welcome

I am a CS PhD candidate at Duke University, advised under Alex Hartemink, specializing in applied machine learning for biological time-series data. Most of my recent work sits in our [lab page](https://github.com/HarteminkLab).
I have ~8+ years developing scalable computational systemsâ€”from real-time mobile applications to advanced genomics frameworks.

## Research & Current Work
My research interests are in applying novel machine learning to interesting problems. For this reason, I've enjoyed working in the genomics space, where large amounts of data abound. These projects require organization and deep understanding of complex biological systems; these are both challenging and rewarding problems.


## Project samples
Here are a few of my projects. For my work, it is important to create detailed, but clear figures.

### [Temporal deconvolution for cell cycle chromatin analysis](https://github.com/HarteminkLab/cell-cycle-deconv/)
A framework for deconvolving the cell cycle chromatin.
<div align="center">
	<img src="https://github.com/HarteminkLab/cell-cycle-deconv/blob/develop/figures/Figure1_Deconvolution.jpg" width="70%"/>
</div>

### [Vision transformer models for gene expression prediction](https://github.com/HarteminkLab/chromatin-transformers)
A vision transformer model to engineer features of the chromatin linked to gene expression. Below are learned attentions that describe which features are most informative for learning predictive features of the chromatin.
<div align="center">
	<img src="https://github.com/HarteminkLab/chromatin-transformers/blob/main/figures/3_learned_attentions.jpg" width="70%"/>
</div>


### [A study of chromatin and transcriptional dynamics under cadmium stress](https://github.com/HarteminkLab/cadmium-paper)
In this work, I analyze a time course of the effects of heavy metals stress on yeast. The strong transcriptional response informed a Gaussian process predictive model.
